ui/src/openapi/openapi.yaml

```diff
@@ -0,0 +1,2324 @@
+---
+# A copy of OpenAPI definition of Nessie 0.44.0, a.k.a. API v1
```

Comment:
shouldnt the filename reflect this i.e. `openapi-nessie044.yaml` or the like?
imo then its clearer this is a temporary thing potentially, not the "leading" openapi file

Reply:
renamed